3-extra/1-guess-the-number-project/script.js

```diff
@@ -20,8 +51,16 @@ function guessNumber() {
 function newGame() {
   //Your code here
   //Reset randomNumber
+  randomNumber = Math.floor(Math.random() * 100 + 1);
```

Comment:
Line 54. Methods are amazing when you using it for the right purpose. Continue to practise and be familiar with all methods. Remember one step at a time.